CONTRIBUTING.md

```diff
@@ -270,6 +272,620 @@ mypy cve_bin_tool/
 for someone who is new or are not familiar to python typing here are few resource - 
 [mypy documentation](https://mypy.readthedocs.io/en/stable/index.html), [resource for more understanding](https://www.linode.com/docs/guides/python-static-type-checking-with-mypy/) and its [Quick reference](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html) and [Python typing documentation](https://docs.python.org/3/library/typing.html)
 
+### Running interrogate by itself
+
+`interrogate` is a tool designed to enhance code documentation by identifying missing docstrings within your Python codebase. `interrogate` will tell you which methods, functions, classes, and modules have docstrings, and which do not.
+
+When running `interrogate`, you need to specify the path to the directory or files that you want to check for missing docstrings. 
+
+```bash
+interrogate [PATH]
+```
+```RESULT: PASSED (minimum: 80.0%, actual: 100.0%)```
+
+In pre-commit, we utilize a range of flags such as -vv, -i, -I, -M, -c, -n, -p, -f, -e. When pre-commit is executed, we receive an output resembling the example below:
```

Comment:
```suggestion
In pre-commit, we use the following flags: 

* `-vv` makes interrogate print a report with all relevant functions showing which ones have or are missing docstrings
* `-i` and `-I` ignore __init__ functions and __init__.py files (because writing "initializes $class" repeatedly is boring)
* The flags `-M`, `-n`, `-c` and `-p` exclude other types of functions.  These are intended to reduce the scope of the problem and help us focus on filling in the most important docstrings first.
* FIXME: Describe the -e and -f tags here.  Use https://github.com/econchick/interrogate if you're not sure

To run exactly what you'll get when pre-commit runs, you can use the following:

FIXME: add exactly the interrogate line you'd use with the correct flags so you can cut and paste it into the console

You can specify a file or directory if you just want to check the files you changed.  

```

This might need some markdown/grammatical fixes as well as filling in the blanks where I left FIXME signs, but this is more what I had in mind.  I added some editorializing on *why* we have things disabled which is probably one of those things that seems obvious to me but maybe isn't obvious to new contributors (so don't feel bad if you hadn't figured out that's why we use all those flags.  that's why it goes in the docs!)